eventuate-core/src/main/scala/com/rbmhtechnology/eventuate/EventsourcedView.scala

```diff
@@ -417,10 +426,19 @@ trait EventsourcedView extends Actor with Stash {
       events.foreach(receiveEvent)
       replay(progress + 1L)
     }
-    case ReplayFailure(cause, iid) => if (iid == instanceId) {
-      logger.error(cause, s"replay failed, stopping self")
-      Try(onRecovery(Failure(cause)))
-      context.stop(self)
+    case ReplayFailure(cause, from, iid) => if (iid == instanceId) {
+      val remainingAttempts = replayAttempts - 1
+      if (remainingAttempts < 1) {
+        // all replay attempts exceeded -> stop the actor
+        logger.error(cause, "replay failed ({} attempts exceeded), stopping self", maxReplayAttempts)
+        Try(onRecovery(Failure(cause)))
+        context.stop(self)
+      } else {
+        // retry replay request while decreasing the remaining attempts
+        logger.warning("replay failed - {} attempts remaining [{}]", remainingAttempts, cause.getMessage)
+        context.become(initiating(remainingAttempts))
+        replay(from)
```

Comment:
Whenever Eventuate retries something it also delays the retry by a configurable amount of time. This should be done here as well.